Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN make install
 
 # Fetch umeed binary
 FROM base-builder AS umeed-builder
-ARG UMEE_VERSION=v2.0.0
+ARG UMEE_VERSION=v3.0.0-alpha2
```

Comment:
we will soon have `alpha3`. Do we need to constantly update it? Can we have some sort of latest `main`?

Reply:
yes, it could be main